parlai/crowdsourcing/tasks/turn_annotations_static/launch_config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai_internal.mturk.block_list import WORKER_BLOCK_LIST
```

Comment:
Can't import from internal - perhaps we could do something here like what I'm doing in my human+model turn_annotations PR? `https://github.com/facebookresearch/ParlAI/pull/3006/files#diff-8913dd3c67cf7f4ac722f014ccb33f24R192` That way the user can pass in blocklist(s)

Reply:
Changed this to pull from a launch config file, which can be customized by the user.